samples/matter/common/src/matter_init.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include <app/clusters/network-commissioning/network-commissioning.h>
+#include <lib/core/Optional.h>
+#include <lib/support/Variant.h>
+#include <platform/PlatformManager.h>
+#if CONFIG_CHIP_FACTORY_DATA
+#include <platform/nrfconnect/FactoryDataProvider.h>
+#else
+#include <platform/nrfconnect/DeviceInstanceInfoProviderImpl.h>
+#endif
+
+namespace Nordic
+{
+namespace Matter
+{
+	using ChipCustomInit = CHIP_ERROR (*)(void);
+	struct ChipInitData {
```

Comment:
If we already started using Matter name (e.g. in namespace), shouldn't we drop naming new structures with `Chip`?

Reply:
I don't know myself :) Chip is still used in most of the Matter SDK interfaces. I'm ok with using only Matter in all added code. @Damian-Nordic, @ArekBalysNordic any thoughts?

Reply:
I don't have a strong opinion whether to use `chip` or `matter` (except that it should be consistent in the common module) but I agree that if the namespace is `matter` then it's unnecessary to add either `chip` or `matter` to class or function names.